src/base/_defaults.scss

```diff
@@ -33,6 +33,27 @@ a {
   color: colors.$text-action;
 }
 
+/**
+ * Remove default spacing from block elements we plan on redefining with our
+ * own classes and structure.
+ */
+
+blockquote,
+dl,
+figure,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ol,
+p,
+pre,
+ul {
```

Comment:
Is there a reason we want to override these but not other [block level elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements)?

Reply:
I pulled this list from our existing patterns, which we had pulled from Suit CSS, but I see no reason not to extend this to other elements!

Reply:
(Assuming those other elements _have_ default margin, of course)

Reply:
@Paul-Hebert Thanks for this, I made a few changes:

1. I consolidated all `margin` overrides from this file to the same place.
1. I expanded the list of elements based on browser default stylesheets, excluding elements I don't anticipate we'd ever use. (See comment.)
1. I removed `margin: 0` declarations from components that are redundant. This also makes the `o-rhythm` layout class easier to use since it won't have the whitespace overruled immediately by a nested `c-button` or `c-input`.